Content.Server/Doors/Systems/DoorSystem.cs

```diff
@@ -238,6 +238,8 @@ private void OnEmagged(EntityUid uid, DoorComponent door, ref GotEmaggedEvent ar
             {
                 SetState(uid, DoorState.Emagging, door);
                 PlaySound(uid, door.SparkSound, AudioParams.Default.WithVolume(8), args.UserUid, false);
+                var emagged = new DoorEmaggedEvent(args.UserUid);
```

Comment:
better raise an event on TryUseEmag() than this and scattering a DoorComponent subscription

Reply:
wouldnt having 2 subscriptions on DoorComponent not be allowed

Reply:
what do you mean?

Reply:
I assumed gloves were emags and meant before that you should raise an event on the emag in TryUseEmag() and handle that subscribed to whatever component the gloves had.
BUT i just had a quick glimpse and saw `OnDoorjack`, why not increase the counter there? (after checking if `DoEmagEffect` was handled)

Reply:
oh yeah good point

Reply:
Well, now raising the event here is not necessary is it

Reply:
i thought i removed real

Reply:
changed to raise EmaggedSomethingEvent in generic emag provider